tools/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -33,6 +33,28 @@ object ToolUtils extends Logging {
     df.showString(numRows, 0)
   }
 
+  def parseClusterTags(clusterTag: String): (String, String, String, String) = {
+    // clusterTags will be in this format -
+    // [{"key":"Vendor","value":"Databricks"},
+    // {"key":"Creator","value":"tgraves@nvidia.com"},{"key":"ClusterName",
+    // "value":"job-215-run-1"},{"key":"ClusterId","value":"0617-131246-dray530"},
+    // {"key":"JobId","value":"215"},{"key":"RunName","value":"test73longer"},
+    // {"key":"DatabricksEnvironment","value":"workerenv-7026851462233806"}]
+    // We first sanitize the input by removing brackets, keywords such as key, value
+    // and create an array consisting of cluster tags.
+    val clusterTagsArray = clusterTag.replaceAll("\\[", "").replaceAll("\\]", "").split("}").
```

Comment:
I guess thi sis ok, but the string above just looks like json, can we simplify this by using json parser?

Reply:
Updated this to use json parser. Thanks for the pointer!